test/src/test/java/hudson/tools/InstallerTranslatorTest.java

```diff
@@ -52,7 +52,8 @@
 
     @Issue("JENKINS-23517")
     @Test public void offlineNodeForJDK() throws Exception {
-        Node slave = new DumbSlave("disconnected-slave", null, "/wherever", "1", Node.Mode.NORMAL, null, new JNLPLauncher(), RetentionStrategy.NOOP, Collections.<NodeProperty<?>>emptyList());
+        Node slave = new DumbSlave("disconnected-slave", null, "/wherever", "1", Node.Mode.NORMAL, null, 
+                DefaultComputerLauncherProvider.findDefaultLauncher(DumbSlave.class), RetentionStrategy.NOOP, Collections.<NodeProperty<?>>emptyList());
```

Comment:
I just wanted to get some free coverage of the extension point. But yeah, maybe it is a bad idea